src/frontend/src/components/compatibilityChart.ts

```diff
@@ -1,22 +1,29 @@
 import { html } from "lit-html";
 
-export const compatibilityChart = html`<p>
-    Browser support for WebAuthentication is constantly evolving. Your preferred
-    browser may not support WebAuthentication, or may only support it using a
-    security key. If you run into issues, please try again with one of our
-    recommended browsers.
-  </p>
+export const compatibilityData = {
```

Comment:
Instead of raw HTML this is now "data" that can be used and styled by the page requesting it.